ocaml/idl/datamodel_errors.ml

```diff
@@ -373,14 +373,8 @@ let _ =
     ~doc:"Cannot restore this VM because it would create a duplicate" ();
   error Api_errors.duplicate_mac_seed [ "seed" ]
     ~doc:"This MAC seed is already in use by a VM in the pool" ();
-  error Api_errors.vm_snapshot_with_quiesce_failed [ "vm" ]
```

Comment:
Please leave these error definitions. They are part of the API definition, and API bindings, or software using the bindings, may break if you remove them.